onedal/datatypes/data_conversion.cpp

```diff
@@ -20,14 +20,26 @@
 #include <string>
 
 #include "oneapi/dal/table/homogen.hpp"
-#include "oneapi/dal/table/detail/csr.hpp"
 #include "oneapi/dal/table/detail/homogen_utils.hpp"
 
 #include "onedal/datatypes/data_conversion.hpp"
 #include "onedal/datatypes/numpy_helpers.hpp"
+#include "onedal/version.hpp"
+
+#if ONEDAL_VERSION <= 20230100
+    #include "oneapi/dal/table/detail/csr.hpp"
+#else
+    #include "oneapi/dal/table/csr.hpp"
+#endif
 
 namespace oneapi::dal::python {
 
+#if ONEDAL_VERSION <= 20230100
```

Comment:
```suggestion
#if ONEDAL_VERSION <= 20230101
```

It should depend on 2023.01.01 build since you have pushed updates recently and they'll be only in this version of OneDAL

Reply:
I think this is for the old interface, and should be ok.